shopping-cart/package.json

```diff
@@ -0,0 +1,58 @@
+{
+  "name": "shopping-cart",
+  "version": "0.1.0",
+  "description": "Sample CloudState event sourced shopping cart application",
```

Comment:
CloudState => Cloud**s**tate

Reply:
I've seen both variants in the docs. Is Cloudstate the correct spelling?

Reply:
I think so. LB changed the Marketing term late 2019 from CloudState to Cloudstate. You would find still classes and types named `CloudState` used in camelcase, but if we talk about a `Sample CloudState event sourced shopping cart application` I think we talk about Cloudstate. Probably not super important, but being consistent helps.

Reply:
the relevant PR https://github.com/cloudstateio/cloudstate/pull/131